generators/client/templates/angular/src/main/webapp/app/shared/language/_translate-partial-loader.provider.ts

```diff
@@ -31,8 +31,12 @@ export class TranslatePartialLoader implements TranslateLoader {
         return Observable.create(observer => {
             this.http.get(`${this.prefix}/${lang}/${part}${this.suffix}`).subscribe((res) => {
                 let responseObj = res.json();
-                Object.keys(responseObj).forEach(key => {
-                    combinedObject[key] = responseObj[key];
+                Object.keys(responseObj).forEach(key=>{
+                    if(!combinedObject[key]) {
+                        combinedObject[key] = responseObj[key];
+                    }else{
+                        $.extend(true,combinedObject[key], combinedObject[key], responseObj[key])
```

Comment:
oh ok I get it. I think I didn't test the scenario. OK but still i would like to do it without jquery if possible. could you try to use `Object.assign` https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign